tests/vm_test.go

```diff
@@ -1412,6 +1412,89 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				})
 			})
 
+			Context("Using RunStrategyOnce", func() {
+				It("[Serial] Should leave a failed VMI", func() {
+					By("creating a VM with RunStrategyOnce")
+					virtualMachine := newVirtualMachineWithRunStrategy(v1.RunStrategyOnce)
+
+					By("Waiting for VM to be ready")
+					Eventually(func() bool {
+						virtualMachine, err = virtClient.VirtualMachine(virtualMachine.Namespace).Get(virtualMachine.Name, &k8smetav1.GetOptions{})
+						Expect(err).ToNot(HaveOccurred())
+						return virtualMachine.Status.Ready
+					}, 360*time.Second, 1*time.Second).Should(BeTrue())
+
+					vmi, err := virtClient.VirtualMachineInstance(virtualMachine.Namespace).Get(virtualMachine.Name, &k8smetav1.GetOptions{})
+					Expect(err).ToNot(HaveOccurred())
+
+					By("killing qemu process")
+					err = pkillAllVMIs(virtClient, vmi.Status.NodeName)
```

Comment:
What about killing the pod instead of this? Then you can make this parallel.

Reply:
/hold

Reply:
killing the pod causes normal shutdown proceedure, which results in phase=succeeded. that's what I tried before pkill

Reply:
Then we could do one more thing `Exec`. But I think this can be tried in follow up :)